### ISSUE TITLE
Improve removing invited user

### ISSUE THREAD

**Issue by Stojdza** [Automattic/wp-e2e-tests#1479]
At some point, test `Inviting New User as a Viewer of a WordPress.com Private Site: @parallel` started to fail because there are too many _Viewers_ for a private site ([example](https://circleci.com/gh/Automattic/wp-e2e-tests/22750)). I tried to reproduce it locally and found that if there are 21 or more viewers test will fail (both, desktop and mobile). Also, it started to fail on [CircleCI](https://circleci.com/gh/Automattic/wp-e2e-tests/22760). 

It looks like that invited user is not on the list, but it is and it's at the bottom of the list. I assume that we have some problem with scrolling and getting that element. 

In test, we have `removeUserByName()` in two places - in [step](https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-invite-users-spec.js#L284) and in [after](https://github.com/Automattic/wp-e2e-tests/blob/master/specs/wp-invite-users-spec.js#L309). @alisterscott I'm guessing that second one is here just to be sure that user will be removed?

**Comment by alisterscott** [comment]
> I'm guessing that second one is here just to be sure that user will be removed?

I think so, maybe we could clean it up though?

We can also try scrolling although it sounds like it may not work well - is there a search function for invited viewers?

**Comment by Stojdza** [comment]
Nope, there is no search option. I'll go with scrolling and see what will happen.
